Tee'sApp/Controllers/ViewController.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  ViewController.swift
+//  Tee'sApp
+//
+//  Created by La Pommeray, Thalia on 10/25/22.
+//
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDataSource{
+    
+    @IBOutlet weak var tableView: UITableView!
+        
+    @IBOutlet weak var searchField: UITextField!
+    
+    @IBOutlet weak var apiActivityIndicator: UIActivityIndicatorView!
+    
+    
+    var data = ["butterfly", "dog", "cat", "mouse", "rabbit", "bat", "chimpanzee", "donkey",
+                "dinosaur", "duck", "eagle", "goldfish"]
+    let activityIndicator = UIActivityIndicatorView()
+    var userName:String = ""
+    var viewModel = InfoViewModel()
+    var userDataHolder: [User] = []
+    var userData: [User] = []
+    var userIndex: Int = 0
+    var user = User.dummyUser()
+    
+
+    override func viewDidLoad() {
+
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+
+        tableView.delegate = self
+        tableView.dataSource = self
+        viewModel.delegate = self
+
+        //delegating search field so a keyboard push registers as the 'return; key
+        searchField.delegate = self
+        //networking data(:
+        viewModel.fetchData()
+        apiActivityIndicator.startAnimating()
+    }
+    
+    
+    @IBAction func searchButtonPressed(_ sender: UIButton) {
+        searchField.endEditing(true)
+        userName = searchField.text!
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        guard let landingVC = segue.destination as? ViewControllerB else{
+            return
+        }
+        
+        //data we plan to pass to our landing ViewController
```

Comment:
Small recommendation: This could be a great opportunity to group these variables into a single data model.  That way, you are only passing a single object to your `landingVC` (making things cleaner and cutting down on the lines of code within your VC)